packages/ui-extensions/src/surfaces/checkout/components/Stack/examples/basic-stack.example.html

```diff
@@ -1,5 +1,4 @@
 <s-stack direction="block" gap="base">
-  <s-box padding="base" borderWidth="base">Box</s-box>
-  <s-box padding="base" borderWidth="base">Box</s-box>
-  <s-box padding="base" borderWidth="base">Box</s-box>
+  <s-box padding="base" borderWidth="base">Free</s-box>
+  <s-box padding="base" borderWidth="base">50% off</s-box>
```

Comment:
Based on the new image, these should be badges.

```suggestion
  <s-badge>Free</s-badge>
  <s-badge>50% off</s-badge>
```

Reply:
Images now :D